src/localplayer.h

```diff
@@ -145,7 +145,7 @@ class LocalPlayer : public Player
 
 private:
 	// clang-format off
-	void accelerateHorizontal(const v3f &target_speed, f32 max_increase, bool slippery);
+	void accelerateHorizontal(const v3f &target_speed, f32 max_increase);
```

Comment:
if you remove this parameter LINT should be okay now, you can remove clang-format markers